.github/workflows/build.yml

```diff
@@ -0,0 +1,95 @@
+name: Build
+
+on:
+  push: ~
+  pull_request: ~
+  release:
+    types: [created]
+  schedule:
+    -
+      cron: "0 1 * * 6" # Run at 1am every Saturday
+
+jobs:
+  tests:
+    runs-on: ubuntu-20.04
+
+    name: "PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [7.4, 7.3, 7.2]
+        symfony: [4.4.*]
+        node: [10.x]
+        mysql: [5.7, 8.0]
+
+        exclude:
+          -
+            php: 7.3
+            mysql: 8.0
+
+    env:
+      ECS_VERSION: 7:0
```

Comment:
Not sure about this one because in the original travis file there were two ECS_VERSION and I don't see how I could do that here in env